packages/common/lib/common/tools.js

```diff
@@ -3439,6 +3439,29 @@ async function getInstancesOrderedByStartPrio(objects, logger, logPrefix = '') {
  * @returns {Promise<void>}
  */
 async function setExecutableCapabilities(execPath, capabilities, modeEffective, modePermitted, modeInherited) {
+    // if Docker and Admin Capabilities should be set check if we are allowed to do that
+    if (isDocker() && capabilities.includes('cap_net_admin')) {
+        try {
+            const systemCaps = fs.readFileSync('/proc/$$/status', 'utf-8');
+            const capBnd = systemCaps.match(/^CapBnd: (.+)$/gm);
```

Comment:
Afaik with the g flag we will not get the array you would expect, also I would repalce the whitespace with \s because it is most likely a tab or something like this